index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 var execall = require('execall');
-var re = /^\s*[^$]\s*(?:-([a-z-]), )?--([a-z-]+) +(.*)$/igm;
+var re = /^\s*[^$]\s*(?:-([a-z-]),\s+)?--([a-z-]+) +(.*)$/igm;
```

Comment:
Either way, what do you prefer ?

The intent here is to match both space and tab I guess.

Something like `[ \t]+` ?